File: Client/app/src/main/java/DTOs/PaymentForAssignDTO.java
package DTOs;

import java.util.ArrayList;

public class PaymentForAssignDTO {
    private int paymentId;
    private ArrayList<Integer> userIds;

    public PaymentForAssignDTO(int paymentId, ArrayList<Integer> userIds) {
        this.paymentId = paymentId;
        this.userIds = userIds;
    }
}

File: Server/DTOs/PaymentForAssignDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Server.DTOs
{
    public class PaymentForAssignDTO
    {
        public int PaymentId { get; set; }
        public ICollection<int> UserIds { get; set; }
    }
}